src/core_modules/capture-core/components/WidgetProfile/DataEntry/buildForm.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import { ProgramFactory } from '../../../metaDataMemoryStoreBuilders/programs/factory/program/ProgramFactory';
+
+const objToMap = (object: any) => {
+    const map = new Map<string, any>();
+    Object.entries(object).forEach((item: any) => {
+        map.set(item[1].id, item[1]);
+    });
+    return map;
+};
+
+const asyncForEach = async function (callback) {
+    for (let index = 0; index < this.length; index++) {
+        // eslint-disable-next-line no-await-in-loop
+        await callback(this[index], index, this);
+    }
+};
+
+const convertProgramForEnrollmentFactory = (program) => {
+    const { programTrackedEntityAttributes, programStages } = program;
+    let convertedProgramTrackedEntityAttributes = programTrackedEntityAttributes.map(programAttribute => ({
+        allowFutureDate: programAttribute.allowFutureDate,
+        displayInList: programAttribute.displayInList,
+        mandatory: programAttribute.mandatory,
+        renderOptionsAsRadio: programAttribute.renderOptionsAsRadio,
+        searchable: programAttribute.searchable,
+        trackedEntityAttributeId: programAttribute.trackedEntityAttribute && programAttribute.trackedEntityAttribute.id,
+    }));
+    let convertedProgramStages = programStages;
+
+    if (!convertedProgramTrackedEntityAttributes.asyncForEach) {
+        convertedProgramTrackedEntityAttributes = [...convertedProgramTrackedEntityAttributes, asyncForEach];
+    }
+
+    if (!convertedProgramStages.asyncForEach) {
+        convertedProgramStages = programStages.map(programStage => ({
+            ...programStage,
+            programStageDataElements: { ...programStage.programStageDataElements, asyncForEach },
+        }));
+        convertedProgramStages = [...convertedProgramStages, asyncForEach];
+    }
```

Comment:
This hack is needed for the Cypress tests. I could not found why but `asyncForEach` property from Array prototype is undefined. This error is happening only in Cypress, localhost works ok. 
Let me know if you have another idea on how to fix it. 
Thank you!